src/Scene/Quadtree.js

```diff
@@ -12,14 +12,39 @@
  */
 define('Scene/Quadtree', [
     'Scene/Layer',
+    'Core/Commander/InterfaceCommander',
     'Core/Geographic/Quad',
     'Renderer/NodeMesh'
-], function(Layer, Quad, NodeMesh) {
-
+], function(Layer, InterfaceCommander, Quad, NodeMesh) {
+
+
+    function commandQueuePriorityFunction(cmd) {
```

Comment:
Makes sense but I would keep it in `Quadtree` until we have another implementation (Octree maybe?) that uses NodeProcess and this priority function.
WDYT?